tests/sqlparser_common.rs

```diff
@@ -11125,16 +11125,9 @@ fn parse_trailing_comma() {
     );
     trailing_commas.verified_stmt(r#"SELECT "from" FROM "from""#);
 
-    // doesn't allow any trailing commas
+    // doesn't allow all trailing commas
     let trailing_commas = TestedDialects::new(vec![Box::new(GenericDialect {})]);
 
-    assert_eq!(
```

Comment:
could we instead change this to use a dialect like postgres that doesnt support trailing commas? thinking that would let us keep the negative test around.

Reply:
Of course, good idea. Implemented in https://github.com/apache/datafusion-sqlparser-rs/pull/1921/commits/83812914bd4f35c6d98b8d9af2ba5d24693492f3